pkg/deploy/install.go

```diff
@@ -259,15 +263,21 @@ func (deploy *Deployer) installMeteringClusterRole() error {
 
 	res.Name = deploy.config.Namespace + "-" + res.Name
 
-	_, err := deploy.client.RbacV1().ClusterRoles().Get(res.Name, metav1.GetOptions{})
+	clusterRole, err := deploy.client.RbacV1().ClusterRoles().Get(res.Name, metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
 		_, err := deploy.client.RbacV1().ClusterRoles().Create(res)
 		if err != nil {
 			return err
 		}
 		deploy.logger.Infof("Created the metering cluster role")
 	} else if err == nil {
-		deploy.logger.Infof("The metering cluster role already exists")
+		clusterRole.Rules = res.Rules
```

Comment:
We could do something like a deep equality check between the current `ClusterRole` object and the current Metering ClusterRole manifest (that gets encoded as a Go object), but I don't think that's worth the cost. This would instead always swap what's present in the YAML manifest with the existing Metering `ClusterRole` k8s object.

Reply:
We may also just want to handle this as we do with the cluster role binding update and always pass the `res` variable (the YAML manifest -> Go object) and call the `.Update()` method.